how-tos/partner-repacks.rst

```diff
@@ -0,0 +1,44 @@
+Partner Repacks
+===============
+
+Partner Repacks are where we take a built and signed Firefox desktop build and modify things (branding, plugins, prefs), potentially also do so for l10n repacks, then re-sign the new files. The main point of contact here will be Mike Kaply.
+
+EME Free is a repacked Firefox without Encrypted Media Extensions, which allows you to run a Firefox without DRM (at the cost of broken video streaming).
```

Comment:
"For example, ..." might make this less awkward.

Reply:
Maybe? EME free is lumped together with partner repacks, but isn't strictly a partner repack.

Reply:
I adjusted the title to make more sense.